rfc/php_technical_committee.wiki

```diff
@@ -136,27 +134,28 @@ The RFC process will continue to be used for process changes, including changes
 === Technical changes ===
 
 As noted, the TC has no special role or authority on user-facing changes.  The TC may, however, decide on the
-implementation of those features. In other words the TC can decide about implementation and ABI but not about API.
+implementation of those features. In other words the TC can decide about implementation and C ABI and API but
+not about PHP API.
 
 The TC SHOULD NOT block the merging of a user-facing change approved by the RFC process, unless the provided
 implementation would result in introduction of new bugs, side effects not mentioned in the RFC, significant performance
-penalties, or if there is an equivalent implementation in progress that TC finds more appropriate.
+penalties not mentioned in RFC, or if there is an equivalent implementation in progress that TC finds more appropriate.
 
 The TC SHOULD give due consideration and weight to previous decisions regarding technical aspects, either by the TC
 or by RFCs prior to the TC's creation.  However, it is NOT REQUIRED to abide by them, and may make decisions as it
 sees best to ensure the quality and maintainability of the code at the time the decision is made.
 
 Technical aspects on which the TC may make a decision include, but are not limited to:
 
-  * Whether a given PR implements a feature or resolves a bug.
+  * Whether a given pull request implements a feature or resolves a bug.
   * Whether a given change qualifies as user-facing, and thus requires an RFC.
   * Whether a given technical aspect or implementation approach is acceptable.
 
 === TC voting ===
 
 Once an issue has been brought to the TC, the Secretary must allow all TC members time to offer their input within one
 week.  Once the Secretary feels the issue has been sufficiently discussed, the Secretary will call a vote of the TC
-members. The vote may be public or private and conducted however the TC chooses.
+members. The vote may be private and conducted however the TC chooses.
```

Comment:
"may be private" still implies it could be public, but not explicitly.  If the intent here is to allow either, it should explicitly say either (as the original text did).  If the goal is to be always-private, then it should read "the vote will be private and...".

I'm good either way.

Reply:
Makes sense - changed to private only